### PR TITLE
2 byte pin index #4097

### PR DESCRIPTION
**Pull request by rusefillc**
Code generator was improved in master, making pin two byte should be a trivial affair now

### CODE REVIEW

File: firmware/config/boards/hellen/cypress/config/controllers/algo/auto_generated_enums.cpp [hunk 1]
```diff
@@ -82,6 +82,18 @@ case EFI_ADC_NONE:
 }
 const char *getBrain_pin_e(brain_pin_e value){
 switch(value) {
+case CAN_PIN_0:
```
Comment (mck1117) [inline firmware/config/boards/hellen/cypress/config/controllers/algo/auto_generated_enums.cpp:85]: can we keep this unrelated-ish change out for now?

File: firmware/integration/rusefi_config.txt [hunk 2]
```diff
@@ -86,13 +86,13 @@
 ! so not forget to change fileVersion in rusefi.ini
 ! todo: this not needed in light of TS_SIGNATURE but rusEFI console still uses it. Need to migrate
 ! rusEFI console from TS_FILE_VERSION to TS_SIGNATURE :(
-#define TS_FILE_VERSION 20200310
+#define TS_FILE_VERSION 20210310
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt:89]: 2021 03 10?

File: firmware/integration/rusefi_config.txt [hunk 3]
```diff
@@ -1092,10 +1087,11 @@ bit useFixedBaroCorrFromMap
 bit useSeparateAdvanceForCranking,"Table","Fixed (auto taper)";+In Constant mode, timing is automatically tapered to running as RPM increases.\nIn Table mode, the "Cranking ignition advance" table is used directly.
 bit useAdvanceCorrectionsForCranking;+This enables the various ignition corrections during cranking (IAT, CLT, FSIO and PID idle).\nYou probably don't need this.
 bit flexCranking;Enable a second cranking table to use for E100 flex fuel, interpolating between the two based on flex fuel sensor.
-bit unused1476b20
 bit useIacPidMultTable;+This flag allows to use a special 'PID Multiplier' table (0.0-1.0) to compensate for nonlinear nature of IAC-RPM controller
 bit isBoostControlEnabled
 bit launchSmoothRetard;+Interpolates the Ignition Retard from 0 to 100% within the RPM Range
+bit unused1476b20
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt:1093]: huh?

Comment (rusefillc) [reply firmware/integration/rusefi_config.txt:1093]: emphasizing that unused bits are available but was moving toward the end

File: firmware/config/boards/hellen/cypress/config/controllers/algo/auto_generated_enums.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/config/boards/hellen/cypress/config/controllers/algo/auto_generated_enums.cpp:85]:
```
enum class MyBigEnum : uint16_t {
    StoredIn16BitsEvenThoughOneValue = 4,
};
```

Comment (mck1117) [reply firmware/config/boards/hellen/cypress/config/controllers/algo/auto_generated_enums.cpp:85]: let's see how https://github.com/rusefi/rusefi/pull/4104 goes

**Pull request merged**